omod/src/main/java/org/openmrs/module/adminui/page/controller/myaccount/ChangeDefaultsPageController.java

```diff
@@ -68,6 +79,35 @@ public UserDefaults getUserDefaults(@RequestParam(value = "defaultLocale", requi
     	return new UserDefaults(defaultLocale, proficientLocales);
     }
 
+    public static Locale normalizeLocale(String localeString) {
```

Comment:
Why do we need custom logic here, as opposed to just doing a one-liner? (Whatever that is should be javadoc commented here.)